src/pages/harvesttype.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <q-page padding>
+      <q-btn
+      padding="xl"
+      color="secondary"
+      round
+      icon="eco"
+      :to="{ path: 'croptype' }"
+    />
+
+    <q-btn
+      padding="xl"
+      color="secondary"
+      round
+      icon= "eco"
```

Comment:
Can you change the icon to something of the meat variant? Or the usual livestock type of icons is fine. So the default material-icons is not enough. So you'll need to update `quasar.conf.js` to enable `mdi-v5` by uncommenting it (remove the `//`). Then you can use the `mdi` icons.